src/services/codefixes/convertToEs6Module.ts

```diff
@@ -303,7 +300,27 @@ namespace ts.codefix {
         return makeExportDeclaration([createExportSpecifier(/*propertyName*/ undefined, "default")], moduleSpecifier);
     }
 
-    function convertExportsDotXEquals(name: string | undefined, exported: Expression): Statement {
+    function convertExportsDotXEquals({ left, right, parent }: BinaryExpression & { left: PropertyAccessExpression }, sourceFile: SourceFile, changes: textChanges.ChangeTracker): void {

```

Comment:
Could "ExportsDotXEquals" be rephrased as "ExportsPropertyAssignment"?

Reply:
:+1: